src/lib/containers/access_requirement_list/AccessRequirementList.stories.tsx

```diff
@@ -0,0 +1,270 @@
+NoRequirements.args = {
+  entityId: mockFileEntity.id,
+  renderAsModal: true,
+}
+
+export const HasMetRequirements = Template.bind({})
+
+HasMetRequirements.parameters = {
+  msw: {
+    handlers: [
+      ...getHandlers(MOCK_REPO_ORIGIN),
+      ...getAccessRequirementHandlers([
+        {
+          accessRequirement: mockManagedACTAccessRequirement,
+          status: {
+            accessRequirementId: mockManagedACTAccessRequirement.id.toString(),
+            concreteType:
+              'org.sagebionetworks.repo.model.dataaccess.ManagedACTAccessRequirementStatus',
+            isApproved: true,
+            expiredOn: '2040-06-01T00:00:00.000Z',
+            currentSubmissionStatus: SubmissionState.APPROVED,
+          },
+        },
+      ]),
+    ],
+  },
+}
+
+HasMetRequirements.args = {
+  entityId: mockFileEntity.id,
+  renderAsModal: true,
+}
+
+export const HasUnmetRequirements = Template.bind({})
+
+HasUnmetRequirements.parameters = {
+  msw: {
+    handlers: [
+      ...getHandlers(MOCK_REPO_ORIGIN),
+      ...getAccessRequirementHandlers([
+        {
+          accessRequirement: mockManagedACTAccessRequirement,
+          status: {
+            accessRequirementId: mockManagedACTAccessRequirement.id.toString(),
+            concreteType:
+              'org.sagebionetworks.repo.model.dataaccess.ManagedACTAccessRequirementStatus',
+            isApproved: false,
+          },
+        },
+        {
+          accessRequirement: mockSelfSignAccessRequirement,
+          status: {
+            accessRequirementId: mockSelfSignAccessRequirement.id.toString(),
+            concreteType:
+              'org.sagebionetworks.repo.model.SelfSignAccessRequirement',
+            isApproved: false,
+          },
+        },
+      ]),
+      rest.post(
+        `${MOCK_REPO_ORIGIN}${ACCESS_APPROVAL}`,
+        async (req, res, ctx) => {
+          const response: AccessApproval = {
+            submitterId: MOCK_USER_ID.toString(),
+            accessorId: MOCK_USER_ID.toString(),
+            state: ApprovalState.APPROVED,
+          }
+          return res(ctx.status(201), ctx.json(response))
+        },
+      ),
+
+      rest.post(
+        `${MOCK_REPO_ORIGIN}${RESEARCH_PROJECT}`,
+        async (req, res, ctx) => {
+          return res(ctx.status(201), ctx.json({}))
+        },
+      ),
+      rest.get(
+        `${MOCK_REPO_ORIGIN}${ACCESS_REQUIREMENT_RESEARCH_PROJECT_FOR_UPDATE(
+          ':id',
+        )}`,
+        async (req, res, ctx) => {
+          const response: ResearchProject = {
+            id: '1981321',
+            accessRequirementId: req.params.id.toString(),
+            institution: 'Black Mesa Research Facility',
+            projectLead: 'Gordon Freeman',
+            intendedDataUseStatement:
+              'We plan to use this data to investigate the properties of antimatter. Our findings will be published in' +
+              ' a peer-reviewed journal, and derivative data will be shared on Synapse.',
```

Comment:
In the past I've add numerous Half Life (and Portal, and other) references in the SWC codebase (in tests) :)